DeeployTest/generateNetwork.py

```
@@ -57,98 +60,166 @@
+            # Load as float64 and infer types later
+            test_inputs = [inputs[x].reshape(-1).astype(np.float64) for x in inputs.files]
+            test_outputs = [outputs[x].reshape(-1).astype(np.float64) for x in outputs.files]
+
+            # WIESEP: Hack to get CI running because only one specific array is used
+            if "WaveFormer" in args.dir:
+                test_inputs = [test_inputs[0]]
+                test_outputs = [test_outputs[-2]]
+
+        platform, signProp = mapPlatform(args.platform)
+
+        for index, (name, num) in enumerate(zip(inputs.files, test_inputs)):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)